### DISCUSSION TITLE
Lesson 2.4 Quiz Question3

### DISCUSSION THREAD

**Discussion by Kiaka007** [OpenMined/courses#179] [Feedback] [answered]
**Question:** If I had a list of whether people like movies - and I replace their names with random names - is it still private/sensitive data?
**Correct Answer:** Yes. Correct - it's still private because data anonymization doesn't work.

I could be reading this wrong but wouldn't this mean that it "No" - it isn't private because anonymization doesn't work?

Same with Question 4.
**Question:** If I can't link anonymized data to someone's exact identity - is it still private information?
**Correct Answer Shown:** Yes, it could be. Correct - because just because you can't identify an individual - if you could identify a community you can still use that data to cause harm.

Wouldn't it be the opposite. So "No" it isn't private because you can use group data in a harmful way?

**Comment by havyarimanaj** [comment]
Hi, I think one way to get clarity could be to look from 2 perspectives: considering (1) the problem to solve : copy and bundling problem and (2) solutions in structured transparency: input privacy, output privacy, input verification, output verification.

**Comment by em-blue** [comment, marked as answer]
Hey @Kiaka007 haha this took me a while to understand what you mean but you're right, the word "private" here could be interpreted to mean two things:
1. "is still private data"= still contains private info that needs to be protected and could cause harm
2. "is still private data"= is protected/ is kept private

We mean number 1! I'll go back and change the wording in those questions to make sure it's crystal clear - thanks for pointing that out cause I would've never seen it from that angle but number 2 makes perfect sense

**Comment by em-blue** [comment]
I've fixed the first one to say 
"Correct - this data could still contain private information, be used to infer private information, or be used to reveal private information contained in other datasets."

And I've changed the next one to say:
If I can't link anonymized data to someone's exact identity - could it still be considered sensitive/private information?
And: "Correct - even if you can't identify an individual if you could identify a community, or have a way to target that individual, you can still use that data to cause harm."

Let me know if that wording gets rid of the potential double-meaning!

**Reply by Kiaka007** [reply]
💡 Oh my goodness  O_O 💡
2. is exactly how I was interpreting it.  That makes way more sense now that I'm reading it as "is it still sensitive data?". Thanks @em-blue ^_^
